.vscode/settings.json

```diff
@@ -19,5 +19,10 @@
             "url": "/.vscode/schema/locale.json"
         }
     ],
-    "cSpell.words": ["Reificated"]
+    "cSpell.words": ["Reificated"],
+    "i18n-ally.localesPaths": [
+        "packages/dashboard/src/locales",
+        "packages/theme/src/locales",
+        "packages/maskbook/src/plugins/Trader/messages"
```

Comment:
🤔or maybe you should add all locale paths. but I doubt if it works with monorepo well.

Reply:
I upgrade i18n plugin in vscode